src/software/thunderscope/constants.py

```diff
@@ -84,6 +84,11 @@ class EstopMode(IntEnum):
 # the maximum packet / world loss percent indicated by UI
 MAX_ACCEPTABLE_PACKET_LOSS_PERCENT = 30
 
+# maximum acceptable round trip time value in milliseconds
+MAX_ACCEPTABLE_MILLISECOND_ROUND_TRIP_TIME = 200
```

Comment:
I had to temporarily change this for field testing since I was only able to get results of 200ms+ during my testing. I am fairly certain this is a bug.

Reply:
@itsarune has a script for measuring latency that we might be able to use to cross reference your values. Alternatively for quickly getting the RTT, you could use `ping` as well.